projects/pt1/e2e_testing/xfail_sets.py

```diff
@@ -1794,6 +1800,9 @@
     "AtenMatmulQint8VV_basic",
     "AtenMatmulQint8VM_basic",
     "AtenMatmulQint8_basic",
+    "QuantizedReluInt32_basic",
```

Comment:
Is there a reason these tests are newly added but failing?

Reply:
It's just an export error.
```
Exporting the operator 'aten::_make_per_tensor_quantized_tensor' to ONNX opset version 17 is not supported. Please feel free to request support or submit a pull request on PyTorch GitHub: https://github.com/pytorch/pytorch/issues.
```
I could test quantized Relu through onnx with something similar to the quantized_models.py tests if you think it would be worth it to add one. I think the quantized models in this list:

```python
COMMON_TORCH_MLIR_LOWERING_XFAILS = {
    "NativeGroupNormBackwardModule_basic",
    "QuantizedMLP_basic",
    "QuantizedSingleLayer_basic",
    "QuantizedBatchedInputSingleLayer_basic",
    "ReduceMaxAlongDimUnsignedInt_basic",
    "ReduceMinAlongDimUnsignedInt_basic",
    "ElementwiseToDtypeI64ToUI8Module_basic",
}
```
 
fail due to the error:

 ```
 error: unknown: unsupported by backend contract: module initializers
        note: unknown: see current operation: "torch.initialize.global_slots"(%18, %22) <{slotSymNames = [@layers.0._packed_params._packed_params, @layers.2._packed_params._packed_params]}> : (!torch.LinearParams, !torch.LinearParams) -> ()
        note: unknown: this is likely due to InlineGlobalSlots being unable to inline a global slot
```

are likely only because of the linear layers not getting unpacked. So a Relu-only model would probably work.

Reply:
Makes sense. Tracing quantized models always seems to be a problem - I hope that dynamo figures that out.